<?xml version="1.0"?>
<launch>
   <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
    <param name="flip_horizontal" value="false" />
  </node>

  <node name="simple_face_detector" pkg="simple_face_detection" type="detector_cv3.py" output="screen">
    <param name="cascade_classifier" value="$(find simple_face_detection)/config/haarcascade_frontalface_default.xml"/>
    <param name="image_topic_input" value="/usb_cam/image_raw" />
    <param name="image_topic_output" value="/face_detection/image_raw" />
  </node>

 <node pkg="HerkuleX" type="HerkuleX_node" name="HerkuleX_node" args="2" output="screen">
 </node>
 <node pkg="HerkuleX" type="HerkuleX_PanTilt_node" name="HerkuleX_PanTilt_node" args="2" output="screen">
 <!-- args="1": Manual Mode || args="2": Auto Mode-->
 </node>



</launch>
